app/urls.py

```diff
@@ -81,5 +81,17 @@
         'requests/<int:pk>/',
         RequestDetail.as_view(),
         name='request_detail'
-    )
+    ),
+    path('interests/',
+         RequestInterestList.as_view(),
+         name='request_interest_list'
+         ),
+    path('interests/<int:pk>/',
+         RequestInterestDetail.as_view(),
+         name='request_interest_detail'
+         ),
+    path('requests/<int:pk>/interests/',
+         MentorRequestInterest.as_view(),
```

Comment:
I'm not crazy about these namings, I'll probably update them in my refactor pull request

Reply:
lol okay no wahala